target/linux/ramips/image/mt7621.mk

```diff
@@ -333,6 +350,40 @@ define Device/iodata_wn-ax1167gr
 endef
 TARGET_DEVICES += iodata_wn-ax1167gr
 
+define Device/iodata_wn-ax1167gr2
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  UBINIZE_OPTS := -E 5
+  UIMAGE_MAGIC := 0x434f4d42
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 51200k
+  DEVICE_VENDOR := I-O DATA
+  DEVICE_MODEL := WN-AX1167GR2
+  KERNEL_INITRAMFS := \
+    $(KERNEL_DTB) | custom-initramfs-uimage 3.10(XBC.1)b10 | iodata-mstc-header
```

Comment:
We now have a guideline for line breaks: e417ff88f19b
I'm not sure if the guidline also applies to `iodata-mstc-header`...

Reply:
Thank you, I'll follow-up that guideline.